src/styles/main.scss

```diff
@@ -10,17 +13,59 @@ body {
   font-weight: 900;
 }
 
+.game-field {
+  background: #bbada0;
+  border-spacing: 10px;
+  border-radius: 5px;
+  position: relative;
+  width: 350px;
+  height: 350px;
+}
+
 .field-cell {
+  position: absolute;
   background: #d6cdc4;
-  width: 75px;
-  height: 75px;
+  width: $cell-size;
+  height: $cell-size;
   border-radius: 5px;
   color: #776e65;
   box-sizing: border-box;
   text-align: center;
   vertical-align: center;
   user-select: none;
 
+  @for $i from 0 through 15 {
+    &--pos--#{$i} {
+      top: ($cell-size  + $cell-margin) * floor($i / 4) + $cell-margin;
+      left: ($cell-size  + $cell-margin) * ($i % 4) + $cell-margin;
+    }
+  }
+}
+
+.field-tile {
+  position: absolute;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background: yellow;
```

Comment:
The background color is set to yellow, but it is not clear why this color has been chosen. Please add a comment to explain or consider using a variable for colors.